src/main/java/io/confluent/connect/jdbc/dialect/GenericDatabaseDialect.java

```diff
@@ -1640,6 +1640,12 @@ public String buildDropTableStatement(
     return Collections.singletonList(builder.toString());
   }
 
+  @Override
+  public void validateSpecificColumnsTypes(
+          ResultSetMetaData rsMetadata,
+          List<ColumnId> timestampColumns
```

Comment:
If this is specific to timestamps, can we change the name to `validateTimestampColumns`, or if it is for all columns, then make this param name more generic?

Reply:
Thanks for the thoughtful comment! 
I think it should be renamed to `validateTimestampColumns`. 
The pattern here is a TableQuerier uses the dialect classes to verify if the columns selected for the mode (timestamp, incrementing or both) in config are valid against the metadata of the table. So I think the play is to have two no-ops in the `GenericDialectClass`, one `validateTimestampColumns` and one `validateIncrementingColumn` (but we should add the latter when it's needed)

Reply:
Actually I like the original generic name....each dialect might different restrictions for columns and this method in the dialect can have logic to check anything specific to the DB, right?